nats-subscriber/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:8
```

Comment:
Should we use a more recent version of node, or does NATS couple us to a specific version?

Reply:
we can likely use node 11

Reply:
actually, the default is 10.x and we used that.